File: go/chess-board-cell-color/chess-board-cell-color.go
package main

import (
	"fmt"
)

func chessBoardCellColor(cell1, cell2 string)bool{
	counter1, counter2 := 0, 0
	for idx, val := range cell1{
		counter1 += int(val)
		counter2 += int(cell2[idx])
	}
	return counter1 % 2 == 0 && counter2 % 2 == 0 
}

func main(){
	cell1, cell2 := "A1",  "C3"

	fmt.Printf("The cells %s and %s of a standard chess board have the same color: %v", cell1, cell2, chessBoardCellColor(cell1, cell2))
}

File: js/chess-board-cell-color.js
function chessBoardCellColor(cell1, cell2){
    const alpha = Array.from(Array(8)).map((e, i) => i + 65);
    const letters = alpha.map((x) => String.fromCharCode(x));
    const numbers = Array.from(Array(8)).map((e, i) => i + 1);

    let blacks = []

    for(let i = 0;i < numbers.length; i++){
        for(let j = 0; j < letters.length; j++){
            if(i % 2 == 0 && j % 2 == 0){
                blacks.push(letters[j] + numbers[i])
            }else if(i % 2 == 1 && j % 2 == 1){
                blacks.push(letters[j] + numbers[i])     
            }
        }
    }

    let isCell1 = blacks.includes(cell1)
    let isCell2 = blacks.includes(cell2)


    return isCell1 && isCell2

}

let cell1 = "A1", cell2 = "C3"

console.log(`The cells ${cell1} and ${cell2} of a standard chess board have the same color: `, chessBoardCellColor(cell1, cell2) )